core/src/test/scala/unit/kafka/admin/AclCommandTest.scala

```diff
@@ -217,17 +217,17 @@ class AclCommandTest extends QuorumTestHarness with Logging {
     createServer(Some(adminClientConfig))
 
     val appender = LogCaptureAppender.createAndRegister()
-    val previousLevel = LogCaptureAppender.setClassLoggerLevel(classOf[AppInfoParser], Level.WARN)
+    appender.setClassLoggerToWarn(classOf[AppInfoParser])
     try {
       testAclCli(adminArgs)
     } finally {
-      LogCaptureAppender.setClassLoggerLevel(classOf[AppInfoParser], previousLevel)
-      LogCaptureAppender.unregister(appender)
+      appender.close()
     }
-    val warning = appender.getMessages.find(e => e.getLevel == Level.WARN &&
-      e.getThrowableInformation != null &&
-      e.getThrowableInformation.getThrowable.getClass.getName == classOf[InstanceAlreadyExistsException].getName)
-    assertFalse(warning.isDefined, "There should be no warnings about multiple registration of mbeans")
+    val warning = appender.getEvents.stream.filter(e => e.getLevel == Level.WARN.toString &&
+      e.getThrowableClassName.isPresent &&
```

Comment:
`e.getThrowableClassName.filter(_ ==classOf[InstanceAlreadyExistsException].getName).isPresent)`

Reply:
Thanks for the suggestion. Updated it.